utils/crud/backingStore.go

```diff
@@ -0,0 +1,119 @@
+package crud
```

Comment:
nit (that you can absolutely ignore): the Go file name for the claim store is `claimstore.go`, while this is `backingStore.go`. Would it make sense ti change either of them to be consistent?

Reply:
Yes, I think that is an artifact of me waffling around while working on a solution. Let me fix that.

Reply:
I'll switch to use `backingstore` to be consistent with other files in this repo